nixos/modules/services/cluster/kubernetes/kubelet.nix

```diff
@@ -336,7 +336,10 @@ in
           [
             gitMinimal
             openssh
-            util-linux
+            # TODO (#409339): remove this patch. We had to add it to avoid a mass rebuild
+            # for the 25.05 release. Once the staging cycle referenced in the above PR completes,
+            # switch back to plain util-linux.
+            util-linux.withPatches
```

Comment:
Wasn't sure whether to switch this to minimal (there is gitMinimal, above). I guess it could make sense, but I'm also afraid of breaking kubernetes somehow by doing that, and we can now build a patched util-linux (not minimal) without rebuilding the world anyway.